git_machete/git_operations.py

```diff
@@ -645,7 +646,7 @@ def __get_merge_base(self, sha1: FullCommitHash, sha2: FullCommitHash) -> FullCo
             #   then there is exactly one merge-base - the ancestor,
             # * if neither of sha1, sha2 is an ancestor of another,
             #   then none of the (possibly more than one) merge-bases is equal to either of sha1/sha2 anyway.
-            self.__merge_base_cached[sha1, sha2] = FullCommitHash.of(self._popen_git("merge-base", sha1, sha2).rstrip())
```

Comment:
Hmmm one thing: pls update the above comment to include the zero merge-base case 🤔